scripts/convert.js

```diff
@@ -21,11 +21,12 @@ const indicators = Object.entries(INDICATORS).map(([key, indicator]) => ({
 getStdin().then((rawCsv) => {
   csv.parse(rawCsv, { columns: true }, (err, data) => {
     const cities = data.map((row) => {
-      const city = {
-        name: row.Cities,
-        id: kebabCase(row.Cities),
-        indices: {},
-      };
+      const city = CITIES[row.Cities];
+
+      if (!city) throw new Error(`${row.Cities} is not a known smart city`);
+
+      city.name = row.Cities;
```

Comment:
Are we sure the long names in the spreadsheet are the ones we want to surface on the website? If not, we could add in `name` or `longName` to the attributes in `cities.js`, and just use row.Cities as the key to look up (then throw away)

Reply:
Good point, don't know. Every time I ask I get a different answer so maybe we revisit this later.